### PR TITLE
Bug 1373779 - XCUITestCase Nightmode Test For On And Off

### PR DESCRIPTION
**Pull request by ugarlapati**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
NightModeTests.swift | 56 | Limit vertical whitespace to a single empty line. Currently 2. 
NightModeTests.swift | 1 | Files should have consistent header comments. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: XCUITests/NightModeTests.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
+//
+//  NightModeTests.swift
+//  Client
+//
+//  Created by Usha Garlapati on 6/16/17.
+//  Copyright © 2017 Mozilla. All rights reserved.
+//
+
```
Comment (npark-mozilla) [inline XCUITests/NightModeTests.swift]: The header for new test cases should be always the same.  You should use the following:

/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

File: XCUITests/NightModeTests.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    
+    private func checkNightModeOff() {
+        navigator.goto(BrowserTabMenu)
+        app.collectionViews.containing(.cell, identifier:"FindInPageMenuItem").element.swipeLeft()
+        waitforExistence(app.collectionViews.cells["HideNightModeItem"])
+        navigator.goto(BrowserTab)
+    }
+    
+
```
Comment (npark-mozilla) [inline XCUITests/NightModeTests.swift]: remove one line of empty space here

File: XCUITests/NightModeTests.swift [hunk 3]
```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+
```
Comment (npark-mozilla) [inline XCUITests/NightModeTests.swift]: @ugarlapati remove empty line

**Pull request merged**